### PR TITLE
feat: move getFrameworkPath from egg-cluster

### PR DESCRIPTION
**Pull request by popomore**
Ref: https://github.com/eggjs/egg/issues/467

### CODE REVIEW

**Comment by fengmk2** [comment]
我发个 minor

**Comment by fengmk2** [comment]
1.2.0

File: index.js [hunk 1]
```diff
@@ -3,16 +3,23 @@
 const fs = require('fs');
 const path = require('path');
 
+[
+  require('./lib/framework'),
+  { getFrameworkOrEggPath },
+]
+.forEach(obj => Object.assign(exports, obj));
```
Comment (atian25) [inline index.js]: 这个写法好怪

**Comment by fengmk2** [comment]
only support node >= 6

release 2.0.0

**Comment by fengmk2** [comment]
1.2.0 unpublished

**Pull request merged**